content/documentation/pages/5-feature-guides/2-streams/1-deployment-properties.md

```diff
@@ -11,7 +11,18 @@ When deploying a stream, properties fall into two groups:
 - Properties that control how the apps are deployed to the target platform and that use a `deployer` prefix are referred to as _deployer properties_.
 - Properties that control or override how the application behave and that are set during stream creation are referred to as _application properties_.
 
-You need to pick a defined platform configuration where each platform type (`local`, `cloudfoundry` or `kubernetes`) has a different set of possible deployment properties. Every platform has a set of generic properties for `memory`, `cpu`, and `disk` reservations and `count` to define how many instances should be created on that platform. The following image shows the Deploy Stream Definition view, where you can set these properties:
+You need to pick a defined platform configuration where each platform type (`local`, `cloudfoundry` or `kubernetes`) has a different set of possible deployment properties. Every platform has a set of generic properties for `memory`, `cpu`, and `disk` reservations and `count` to define how many instances should be created on that platform.
+
+<!--TIP-->
+
+You can view the deployment properties for each of the platforms by selecting one of the following links: [local](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#configuration-local-deployer), [cloudfoundry](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#configuration-cloudfoundry-deployer) or, [kubernetes](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#configuration-kubernetes-deployer).
+
+<!--END_TIP-->
+
+The following image shows the Deploy Stream Definition view, where you can set these properties:
```

Comment:
The next four lines should be:

You can set the following properties:

* Deployer Properties: These properties customize how tasks are launched.
* Application Properties: These are application specific properties.

The following image shows the Deploy Stream Definition view:

Reply:
Actually this content is a repeat of the start of the section, cut and paste from the corresponding page for tasks. I will clean up.